Makefile

```diff
@@ -70,12 +75,77 @@ init:
 verify-tag-name: print-global-variables
 	# Check for invalid tag name
 	t=${TAG} && [ $${#t} -le 128 ] || { echo "Target name $$t has 128 or more chars"; false; }
+.PHONY: generate-client ## Generate client packages
+generate-client: code-generator
+	rm -rf pkg/client/clientset/versioned pkg/client/informers/externalversions pkg/client/listers/controller/v1beta1
+# TODO: add this back when the version of the tool has been updated and supports this executable
+#	$(APPLYCONFIGURATION_GEN) \
+#		--input-dirs="github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/apis/controller/v1beta1" \
+#		--go-header-file="hack/boilerplate/boilerplate.go.txt" \
+#		--output-package="github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/client/applyconfiguration" \
+#		--trim-path-prefix "github.com/project-codeflare/multi-cluster-app-dispatcher"
+	$(CLIENT_GEN) \
+ 		--input="pkg/apis/controller/v1beta1" \
```

Comment:
Quota Management API package can be added here.

Reply:
Done!